epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityTracker.java

```diff
@@ -100,6 +103,19 @@ public void setOnChangedEnabled(boolean enabled) {
     onChangedEnabled = enabled;
   }
 
+  /**
+   * Set the threshold of percentage visible area to identify the partial impression view state.
```

Comment:
maybe not that the default makes partial impression events never called?

Reply:
adressed also in b6d43a5b10b06c43115a6942fd84d2c9962fffb1